src/test/java/com/trivago/fastutilconcurrentwrapper/AbstractMapTest.java

```diff
@@ -0,0 +1,65 @@
+package com.trivago.fastutilconcurrentwrapper;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+/*
+ *  Since there is no common Map interface for the typed maps, in fastutilsconcurrentwrapper,
+ *  test cases have to be repeated for each typed map interface.
+ */
+public abstract class AbstractMapTest {
+    protected final Random random = ThreadLocalRandom.current();
```

Comment:
If you do that way, you use `Random` related to a thread that instantiates a class.
I would rather use `ThreadLocalRandom.current()` in a method.

Reply:
* Good point. Made `ThreadLocalRandom.current()` to be invoked at method level.